ui/pages/confirmation/templates/index.js

```diff
@@ -22,6 +28,7 @@ const ALLOWED_TEMPLATE_KEYS = [
   'cancelText',
   'onApprove',
   'onCancel',
+  'networkDisplay',
```

Comment:
This is part of a change that makes the network display on these templates optional (because the snap confirmation didn't pertain to any network). We can split out this change.

Reply:
@hmalik88